packages/react-core/src/components/Toolbar/examples/ToolbarColorVariant.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Toolbar, ToolbarItem, ToolbarContent, Button, SearchInput, Checkbox } from '@patternfly/react-core';
+
+export const ToolbarColorVariant: React.FunctionComponent = () => {
+  const [isSecondary, setisSecondary] = React.useState(true);
+
+  return (
+    <React.Fragment>
+      <Toolbar id="toolbar-color-variant" colorVariant={isSecondary ? 'secondary' : 'primary'}>
+        <ToolbarContent>
+          <ToolbarItem>
+            <SearchInput aria-label="Items example search input" />
```

Comment:
Just to give this a unique name:

```suggestion
            <SearchInput aria-label="With secondary color example search input" />
```